invenio_stats/processors.py

```diff
@@ -60,22 +59,39 @@ def anonymize_user(doc):
     if ip:
         doc.update(get_geoip(ip))
 
-    uid = doc.pop('user_id', '')
-    ua = doc.pop('user_agent', '')
+    user_id = doc.pop('user_id', '')
+    session_id = doc.pop('session_id', '')
+    user_agent = doc.pop('user_agent', '')
+    timestamp = arrow.get(doc.get('timestamp')).strftime('%Y%m%d%H')
```

Comment:
Maybe rename this to `timeslice` and explain in a comment what this is (w.r.t. COUNTER terminology)